### DISCUSSION TITLE
Possible typo in the documentation

### DISCUSSION THREAD

**Discussion by andy90** [e3nn/e3nn#307] [Q&A] [answered]
I believe there is a typo in documentation [https://docs.e3nn.org/en/stable/guide/convolution.html#convolution](url). In the first equation, fi on the right hand side of the equation should be replaced by fj. Am I right?

**Comment by mariogeiger** [comment, marked as answer]
Indeed. Thanks!

**Reply by mariogeiger** [reply]
fixed, will appear in the next release
